internal/daemon/controller/handlers/mask_manager.go

```diff
@@ -117,3 +117,12 @@ func MaskContains(paths []string, s string) bool {
 	}
 	return false
 }
+
+func MaskContainsSubString(paths []string, s string) bool {
+	for _, p := range paths {
+		if strings.Contains(p, s) {
```

Comment:
Should be not be doing a `MaskContainsPrefix` and validate the field mask has the prefix of `attributes.object`

Reply:
I like the name `MaskContainsPrefix`. I will apply the change.